force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -321,18 +381,28 @@ public with sharing class FlowControlLoopsCollections {
      * @return The salary of the employee, or null if the employee is not found in the Map.
      */
     public static Integer getSalary(String employeeName) {
+        Integer salary;
         // Initialize a Map
-
+        Map<String, Integer> employees =  new Map<String, Integer> ();
         // Add key-value pairs to the Map
         // add John Doe => 50000
+        employees.put('John Doe', 50000);
         // add Jane Smith => 60000
+        employees.put('Jane Smith', 60000);
         // add Sam Brown => 55000
+        employees.put('Sam Brown', 55000);
         // add Alice Johnson => 65000
+        employees.put('Alice Johnson', 65000);
 
         // Get the salary of the employee
-
+        if(employees.containsKey(employeeName)){
+            salary = employees.get(employeeName);
+        }
+        else{
+            salary = null;
+        }
         // Return the salary of the employee, or null if the employee is not found in the Map
-        return null; // Replace null with the variable you used to store the result
+        return salary; // Replace null with the variable you used to store the result
```

Comment:
I will never fault someone for checking containsKey(). But in this case, if the key were not present, the expression would evaluate to null because there is no value.